app/models/plugins/has_form.rb

```diff
@@ -33,7 +33,7 @@ def dup
     clone = super
     clone.save!
 
-    clone.form.form_elements = form.form_elements.map(&:dup) if clone.form
+    clone.form&.form_elements = form.form_elements.map(&:dup)
```

Comment:
these are not equivalent expressions. If `clone.form` is not present this raises an `Undefined method form elements for nil` exception. We should keep the guard expression.